### PR TITLE
Add Micro npm Packages

### PR DESCRIPTION
**Pull request by parro-it**
https://github.com/parro-it/awesome-micro-npm-packages

Previously discussed [here](https://github.com/sindresorhus/module-requests/issues/56).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 	- [Must Watch Talks](https://github.com/bolshchikov/js-must-watch)
 	- [Tips](https://github.com/loverajoel/jstips)
 	- [Network Layer](https://github.com/Kikobeats/awesome-network-js)
+	- [Micro NPM Modules](https://github.com/parro-it/awesome-microlibs)
```
Comment (sindresorhus) [inline readme.md]: `npm` is lowercase

There's nothing called `NPM Modules`. It's `Node.js modules` and `npm packages`. Different concepts.

Comment (sindresorhus) [inline readme.md]: Your list title is also very vague, `microlibs`. Could be any language/ecosystem.

Comment (parro-it) [inline readme.md]: What do you suggest for the title? What about `awesome-node-micropackages`?

Comment (sindresorhus) [inline readme.md]: They're not really only for Node.js though. Most of them work in the browser too. `awesome-micro-npm-packages`

Comment (parro-it) [inline readme.md]: Cool, I like it! :sunglasses:

**Pull request closed without merging**